__tests__/Port.test.js

```diff
@@ -0,0 +1,57 @@
+const { Port } = require("../src/Port")
+
+describe("Port", () => {
+  let port, ship1, ship2
+
+  beforeEach(() => {
+    port = new Port("Liverpool")
+    ship1 = jest.fn()
```

Comment:
As before why is a function being used to make the mock? It should represent the value that is actually passed in when the constructor is called